tcms/signals.py

```diff
@@ -67,23 +67,23 @@ def notify_admins(sender, **kwargs):
     user = kwargs.get('user')
 
     admin_emails = []
-    for name, email in settings.ADMINS:
+    for _, email in settings.ADMINS:
```

Comment:
Tip: you can also `_name` and leave the `_` function as it is if the function doesn't raise any warnings. Less typing.